spring-kafka/src/test/java/org/springframework/kafka/listener/DefaultErrorHandlerNoSeeksBatchAckTests.java

```diff
@@ -0,0 +1,219 @@
+/*
+ * Copyright 2017-2021 the original author or authors.
```

Comment:
Current year, please:
```suggestion
 * Copyright 2022 the original author or authors.
```